ROMFS/px4fmu_common/init.d/airframes/4400_ssrc_fog_x

```diff
@@ -38,9 +41,6 @@ param set-default RTPS_MAV_CONFIG 0
 # Enable LL40LS in i2c
 param set-default SENS_EN_LL40LS 2
 
-# Set EKF to use range finder as main height source
-param set-default EKF2_HGT_MODE 2
```

Comment:
Do we want to disable the range finder completely in the outdoor missions also? 

During initial tests with fog_navigation, we observed that if this parameter is set to 0 (barometer) in a strong wind with gusts setting, it can cause huge variances in the altitude estimate. Both when the drone is on the ground and during flight. During flight, it was observed that the drone altitude estimates were varying large enough to make the drone unable to reach the goal altitude. 

`EKF2_RNG_A_HMAX = 5.0` already limits the maximum height to use range finder information.

Reply:
No,

See analysis in bug  https://ssrc.atlassian.net/browse/DP-3611

Setting the *main* height source to RNG is good for indoors flying, where it is not supposed to use barometer at all. This setting disables the EKF2_RNG_A_HMAX setting completely, and it only switches to barometer after 5s timeout when it cannot any more receive data from the HGT.

With combination of EKF2_HGT_MODE == BARO and EKF2_RNG_AID=1 it always uses range finder in low speeds and under EKF2_RNG_A_HMAX, but does immediate switch to barometer when speeding or above EKF2_RNG_A_HMAX, which is more correct ouside.

The EKF2_HGT_MODE == RNG it also disables possibility to switch to GPS height as a backup. The crash in the bug mentioned above was partly due to the barometer faling earlier (already before that flight), and no height sources left as GPS height was not usable.

Reply:
You are correct. Maybe I didn't read the documentation that clearly before. I just checked the [Range Aid](https://docs.px4.io/v1.9.0/en/flying/terrain_following_holding.html#range_aid), it describes the behavior clearly. 

So, `EKF2_HGT_MODE` default parameter can be used as you explained. Should we also add `EKF2_RNG_AID = 1` since we are using `EKF2_HGT_MODE == BARO`? Otherwise, it will only use the barometer, which might be the problem during with the flight I explained. It was causing the altitude estimate vary quite largely when the drone is just on the ground (~1meter/minute).

Reply:
Yes, the EKF2_RNG_AID = 1 should be there